tests/test_database_types.py

```diff
@@ -30,6 +30,15 @@
 CONNS[db.MySQL].query("SET @@session.time_zone='+00:00'", None)
 oracle.SESSION_TIME_ZONE = postgresql.SESSION_TIME_ZONE = "UTC"
 
+if db.Exasol in CONNS:
+    kw = dict(CONNS[db.Exasol].kwargs)
+    kw["driver"] = "exasol"
+    kw["host"] = kw.pop("dsn")
+    schema = kw.pop("schema")
+    conn_without_schema = db.connect.connect(kw)
+    conn_without_schema.query(f"CREATE SCHEMA IF NOT EXISTS {schema}", None)
```

Comment:
You should add double quotes around `{schema}` otherwise Exasol will automatically uppercase the provided schemaname in the URI. This would be better: `conn_without_schema.query(f"CREATE SCHEMA IF NOT EXISTS \"{schema}\"", None)` to prevent errors with lowercase schemanames.

Reply:
Feel free to merge it into your PR and fix whatever you need.